analytics/analytics/job_processor/prometheus.py

```diff
@@ -0,0 +1,293 @@
+import math
+import statistics
+from datetime import datetime
+from urllib.parse import urlencode
+
+import requests
+from kubernetes.utils.quantity import parse_quantity
+
+from analytics.models import Job, JobPod, Node
+
+CLUSTER_INTERNAL_PROMETHEUS_URL = (
+    "kube-prometheus-stack-prometheus.monitoring.svc.cluster.local:9090"
```

Comment:
Can this be parameterized?

Reply:
Parameterized in what way? The `PrometheusClient` already takes an argument to specify the url to use, so this just serves as a constant for the value we use in prod.

If you're referring to the fact that this value is hard-coded at all, I agree, however I couldn't find a clear way to retrieve this value from the cluster via secrets, config-maps, etc.

Did you have something specific in mind?

Reply:
Is this value the same in staging and production? If so, I guess this is fine - though it is a shame we can't populate it into the settings somehow.

Reply:
It is the same in prod/staging, as it's just the internal IP to the service. I agree it's a shame.